app/views/collections/index.html.erb

```diff
@@ -54,7 +54,7 @@
   <!--main content-->
   <h3 class="landmark heading"><%= ts("List of Collections") %></h3>
   <ul class="collection picture index group">
-    <% @collections.each do |collection| %>
+    <% @collections.includes(:parent, :moderators, :children, :collection_preference, owners: [:user]).each do |collection| %>
```

Comment:
So in the version Elz reviewed in #2229, the `owners` part is different: `<% @collections.includes(:parent,:owners,:moderators,:children,:collection_preference).each do |collection| %>` I'm also not sure if we want this code in the view or if it would be better off in the controller or a helper, so I'm pinging @elzj for her opinion on both matters here.